bucket/xenia-canary.json

```diff
@@ -5,19 +5,14 @@
     "license": "BSD-3-Clause",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/xenia-canary/xenia-canary/releases/download/0c2f00a/xenia_canary.zip",
-            "hash": "a0c643797cbecc142e997140165d15f6cb3cd7c06bc1ef3d9e502bcec7f4eeea"
+            "url": "https://github.com/xenia-canary/xenia-canary/releases/download/experimental/xenia_canary.zip",
+            "hash": "c5206f7028c7a90be4a6e279a1a16ddbd55c36b168b2e2a3503f5cbe67a1dcab"
```

Comment:
Since the URL is now fixed, the version identifier should be changed to `nightly` as hash verification will not work reliably (the auto-updater may be running too early or too late).

Reply:
They seem to be providing the legacy tags still. I opened an issue with them to see their recommendation.
I can either do these changes now or drop this change from the pr until i hear back from them, but it looks like this would be a more reliable way to get the latest release.

One thing about nightly in scoop I noticed is that it doesn't update it unless you add --force, is that expected?